### ISSUE TITLE
Redo video

### ISSUE THREAD

**Issue by ChrisChinchilla** [SuperblocksHQ/ethereum-studio#161]
Redo video to introduce more basic concepts and remove assumptions.

**Comment by javier-tarazaga** [comment]
@ChrisChinchilla is there any update on this?

**Comment by ChrisChinchilla** [comment]
@javier-tarazaga Yeah I can have another go with this soon. I'm conscious that at the moment you are making a few interface changes, so making a video might be wasted if the interface changes again. When do you think might be a good time to redo? i.e. when will the changes mostly be done for this release?

**Comment by javier-tarazaga** [comment]
@ChrisChinchilla what should we do with this issue? Still up to re-do the video with the current changes in place?

**Comment by ChrisChinchilla** [comment]
Good call. I don't have access to the studio at the moment (unsurprisingly) but i'll redo this sometime in the next week.
